assets/js/trento.jsx

```diff
@@ -67,7 +67,7 @@ function App() {
                 element={<ChecksResults />}
               />
               <Route
-                path="clusters/:clusterID/executions/:executionID"
+                path="clusters/:clusterID/executions"
```

Comment:
Now the path could be just `last_execution` maybe?

Reply:
Yeah sure!